patches/tools-grit-preprocess_if_expr.gni.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/tools/grit/preprocess_if_expr.gni b/tools/grit/preprocess_if_expr.gni
+index 850bff9aa94b4..6af9549d8d843 100644
+--- a/tools/grit/preprocess_if_expr.gni
++++ b/tools/grit/preprocess_if_expr.gni
+@@ -47,6 +47,7 @@ template("preprocess_if_expr") {
+       }
+     }
+ 
++    inputs += ["//brave/script/gen_webui_override_additional_inputs.py"] + exec_script("//brave/script/gen_webui_override_additional_inputs.py", ["--in-folder", rebase_path(in_folder, root_build_dir), "--in-files"] + invoker.in_files, "list lines")
```

Comment:
I suspect this isn't what we want to be doing @bridiver but I was a bit at a loss how to have changes to the `chromium_overrides` trigger a rebuild of the target.

##### Downsides of this approach:
1. New `chromium_overrides` won't trigger a build (though changes to existing ones will).
2. Uses `exec_script` which Upstream warns is slow on windows (and generally a Not Good Thing).
3. Requires patching Upstream's `.gn` file to allow `exec_script`